pages/common/kubectl.md

```diff
@@ -29,3 +29,7 @@
 - Print the address of the master and cluster services:
 
 `kubectl cluster-info`
+
+- For getting explanations of fields:
```

Comment:
Thanks for the edit, @spachava753! Perhaps this will read better?
```suggestion
- Display an explanation of the fields shown:
```

Reply:
I would say: 
`- Get the documentation of the resource and its fields`

Reply:
In this specific example, you will get an output of **_deployment_** documentation. 

For example, if you run: `kubectl explain pods.spec.containers`, then you will get the explanation of that specific field of the resource (`.containers` field)

Reply:
I would suggest:

`- Get the documentation of the resource and its fields:`

Reply:
Ah, I see @Aracki. What about this instead then?

```suggestion
- Show the documentation of the resource and its fields:
```

Reply:
Erm.... never mind. Looks like it's already been changed below.